metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -94,6 +113,12 @@ class RestSinkEndpoint(ConfigEnum):
 )
 
 
+DEFAULT_REST_TRACE_MODE = pydantic.parse_obj_as(
+    RestTraceMode,
+    os.getenv("DATAHUB_REST_TRACE_MODE", RestTraceMode.DISABLED),
```

Comment:
can we add a comment on the minimum backend version supporting this?

alternatively, is this something that we could manage dynamically from `/config`? 
https://github.com/datahub-project/datahub/blob/fa4ff7bced2aca53fbd1571d7a87171b11b0a002/metadata-ingestion/src/datahub/ingestion/graph/client.py#L346

Reply:
Added comment, also added note to release notes that it does require version 1.0